model-customization/data-processing/chunking/docling-chunking.ipynb

```diff
@@ -0,0 +1,236 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "290ef089-c1b5-4a95-8a0e-1c650bac5ea2",
+   "metadata": {},
+   "source": [
+    "# Chunking\n",
+    "\n",
+    "The goal of chunking the converted documents is to provide the teacher model small and logical pieces of the source document to generate data off of.\n",
```

Comment:
Can we assume the user is familiar with SDG so that then know what "generate data off of" means? Maybe just a bit more exposition and/or a link to something around SDG?